src/renderer/components/editable-list/editable-list.scss

```diff
@@ -0,0 +1,12 @@
+.EditableList {
+  .el-contents {
+    display: grid;
+    grid-template-columns: 1fr auto;
+
+    .el-value-remove {
+      .Icon {
+        justify-content: unset;
+      }
+    }
+  }
+}
```

Comment:
In Preferences page we already have some styling for similar component. Can you tweak `EditableList` styles to make them consistent with helm repos in Preferences?

Here is the current view:
<img width="607" alt="Screenshot 2020-09-02 at 10 24 45" src="https://user-images.githubusercontent.com/9607060/91970736-c2bb1b80-ed20-11ea-911e-8927b4613e81.png">

And here is how it should look like 
<img width="621" alt="Screenshot 2020-09-02 at 10 42 25" src="https://user-images.githubusercontent.com/9607060/91970817-e3837100-ed20-11ea-9ee4-3ffbd03c611a.png">